slave/configs.py

```diff
@@ -52,27 +52,25 @@ def prefs(self):
         # Currently only for firefox profile.
         return self.prefs_
 
-class Wasm(Default):
-    def __init__(self, engine, shell):
-        super(Wasm, self).__init__(engine, shell)
-        if engine == "firefox":
-            self.prefs_["javascript.options.wasm"] = True
-        elif engine == "chrome":
-            self.args_ += ['--js-flags=--expose_wasm']
-
-class WasmBaseline(Wasm):
+class WasmBaseline(Default):
     def __init__(self, engine, shell):
         super(WasmBaseline, self).__init__(engine, shell)
         if engine == "firefox":
             self.prefs_["javascript.options.wasm_baselinejit"] = True
             self.prefs_["javascript.options.wasm_ionjit"] = False
```

Comment:
We no longer require self.prefs_["javascript.options.wasm"] = True ?

Reply:
No, I think it should be true by default. Got to check for Beta etc, though...

Reply:
wasm is now enabled even on release.